overview.md

```diff
@@ -19,27 +19,37 @@ TLDR: Push the optimistic-rollup state of the art by leveraging Ethereum Layer 1
 
 ## Components
 
-- **L1**:
-  - Data availability layer (eth1 event logs or sharding):
-    - (Batch) Transactions
-    - State roots
-  - **Bond Contract**
-  - **Dispute Game Contract**:
-      - Bi-section game on merkle tree of execution steps between the trusted start and disputed end
-  - **Verification Engine Contract**
-      - Loads witness data:
-          - step witness parts can be represented as `generalized index -> bytes32`, 
-            to simplify interaction with the partial structure (no tree management, every lookup is a simple math evaluation to get the right key)
-          - involved contract code entries can be large, TBD if put in storage or better of in call-data
-          - MPT nodes are just a dict (of internal MPT node hash to MPT node content)
-      - Runs verification step
-      - Registers result in bond-manager (slash sequencer, reward challenger, or vice-versa if spam attempt)
-  - **Optimistic Bridge**
-    - Log authorized calls as forced L2 transactions
-      - Enables Deposits
-    - Verify L2 data statelessly with MPT proof to undisputed state-root
-      - Enables Withdrawals
-- **Rollup node** (previously "Data Transport Layer"):
+![Architecture Diagram](architecture.svg)
```

Comment:
Could you add link to draw.io source?

Reply:
The drawio svg's have the full XML source as metadata.  So, you can just load the SVG directly in and it will work.

I have also successfully used "open from github" in drawio on this branch--I just had to make a personal fork so as not to have to grant org access.

Reply:
Oh wow that's sweet! 🔥🔥🔥